CMakeLists.txt

```diff
@@ -70,7 +76,7 @@ option(GGML_SANITIZE_ADDRESS   "ggml: enable address sanitizer"   OFF)
 option(GGML_SANITIZE_UNDEFINED "ggml: enable undefined sanitizer" OFF)
 
 # instruction set specific
-if (GGML_NATIVE)
+if (GGML_NATIVE OR NOT GGML_NATIVE_BUILD)
```

Comment:
```suggestion
if (GGML_NATIVE OR CMAKE_CROSSCOMPILING)
```

Reply:
I'd rather have the GGML calculated value here, as `CMAKE_CROSSCOMPILING` may lead to false negatives on some rare occasions (and even false positives on even rarer occasions). In case we ever handle them, `GGML_NATIVE_DEFAULT` will carry the correct information.

Reply:
Sounds good.
```suggestion
if (GGML_NATIVE OR NOT GGML_NATIVE_DEFAULT)
```

Reply:
resolved in force-pushed commit